config/db.sample.js

```diff
@@ -1,6 +0,0 @@
-const config = {
```

Comment:
Why was this file removed? We need this is as a guide for people who want to use this first time.